src/kernel/pgtbl.c

```diff
@@ -1,187 +1,234 @@
+/******************************************************************************
+Filename    : pgtbl.c
+Author      : Gabriel Parmer
+Date        : ?/?/2015
+Licence     : GPL v2; see COPYING for details.
+Description : The page table operations that does not need special optimizations.
+******************************************************************************/
```

Comment:
I thought we agreed on uniformity for comment style/license style. ??

Reply:
Ok I will delete this